common/src/main/java/net/adoptopenjdk/icedteaweb/jvm/JvmUtils.java

```diff
@@ -54,71 +72,188 @@ private static boolean isValidStartingArgument(final String argument) {
-            "-XX:PermSize", /* set initial size of permanent gen */
-            "-XX:MaxPermSize", /* set max size of permanent gen */
-            "-XX:MaxHeapFreeRatio", /* heap free percentage (default 70) */
-            "-XX:MinHeapFreeRatio", /* heap free percentage (default 40) */
-            "-XX:UseSerialGC", /* use serial garbage collection */
-            "-XX:ThreadStackSize", /* thread stack size (in KB) */
-            "-XX:MaxInlineSize", /* set max num of bytecodes to inline */
-            "-XX:ReservedCodeCacheSize", /* Reserved code cache size (bytes) */
-            "-XX:MaxDirectMemorySize",};
+                "-ea", /* enable assertions for classes */
+                "-enableassertions", /* enable assertions for classes */
+                "-da", /* disable assertions for classes */
+                "-disableassertions", /* disable assertions for classes */
+                "-verbose", /* enable verbose output */
+                "-Xms", /* set initial Java heap size */
+                "-Xmx", /* set maximum Java heap size */
+                "-Xss", /* set java thread stack size */
+                "-XX:NewRatio", /* set Ratio of new/old gen sizes */
+                "-XX:NewSize", /* set initial size of new generation */
+                "-XX:MaxNewSize", /* set max size of new generation */
+                "-XX:PermSize", /* set initial size of permanent gen */
+                "-XX:MaxPermSize", /* set max size of permanent gen */
+                "-XX:MaxHeapFreeRatio", /* heap free percentage (default 70) */
+                "-XX:MinHeapFreeRatio", /* heap free percentage (default 40) */
+                "-XX:UseSerialGC", /* use serial garbage collection */
+                "-XX:-UseSerialGC",
+                "-XX:ThreadStackSize", /* thread stack size (in KB) */
+                "-XX:MaxInlineSize", /* set max num of bytecodes to inline */
+                "-XX:ReservedCodeCacheSize", /* Reserved code cache size (bytes) */
+                "-XX:MaxDirectMemorySize", /* set maximum direct memory size */
+                "-XX:PrintCMSStatistics", /* If > 0, Print statistics about the concurrent collections */
+                "-XX:SurvivorRatio",                    /* Sets the ratio of the survivor spaces */
+                "-XX:MaxTenuringThreshold",             /* Determines how much the objects may age */
+                "-XX:CMSMarkStackSize",
+                "-XX:CMSMarkStackSizeMax",
+                "-XX:CMSIncrementalDutyCycleMin",       /* The percentage which is the lower bound on the duty cycle */
+                "-XX:ParallelCMSThreads",
+                "-XX:ParallelGCThreads",                /* Sets the number of parallel GC threads */
+                "-XX:CMSInitiatingOccupancyFraction",   /* Sets the threshold percentage of the used heap */
+                "-XX:+UseCompressedOops",               /* Enables compressed references in 64-bit JVMs */
```

Comment:
Is this really a prefix argument or should this be moved to the valid arguments list?